pkg/cpio/cpio.go

```diff
@@ -81,6 +81,27 @@ func NewDedupWriter(rw RecordWriter) RecordWriter {
 	}
 }
 
+// Passthrough reads in a cpio from a root and writes it to a destination root.
+// This avoids copying the entire partition which is much larger than the cpio.
```

Comment:
why is this second line of the comment here? partition?

Reply:
actually you probably should redo the whole comment.
// Passthrough copies from a RecordReader to a RecordWriter
// It processes one record at a time to minimize the memory footprint.